cms/templates/index.html

```diff
@@ -27,10 +28,17 @@ <h3 class="sr">${_("Page Actions")}</h3>
           % elif course_creator_status=='disallowed_for_this_site' and settings.FEATURES.get('STUDIO_REQUEST_EMAIL',''):
           <a href="mailto:${settings.FEATURES.get('STUDIO_REQUEST_EMAIL','')}">${_("Email staff to create course")}</a>
           % endif
+
           % if show_new_library_button:
             <a href="#" class="button new-button new-library-button"><i class="icon fa fa-plus icon-inline"></i>
             ${_("New Library")}</a>
           % endif
+
+          % if is_programs_enabled:
+            <!-- TODO: Link to the program creation view in the authoring app. -->
+            <a href="#" class="button new-button new-program-button"><i class="icon fa fa-plus icon-inline"></i>
```

Comment:
This should probably be ``button`` too, although I see that there are several other faux buttons above it that use ``a``.